protect-function-app/README.md

```diff
@@ -0,0 +1,121 @@
+| **Token store**                 | _Unselected_                                            | Suggested value for this sample.                                                 |
+
+> :information_source: **Bold text** in the table matches (or is similar to) a UI element in the Azure portal, while `code formatting` indicates a value you enter into a text box in the Azure portal.
+### 3. Deploy the Function app
+
+```console
+func azure functionapp publish <your-Function-App-name>
+```
+
+## Access the API
+
+Using Postman, curl, or a similar application, issue an HTTP GET request to https://<your-function-app-name>.azurewebsites.net/api/greeting with an `Authorization` header of `Bearer {VALID-ACCESS-TOKEN}`.
+
+For example, if you use curl and everything worked, you should receive a response from the Azure Function similar to this.
+
+```console
+$ curl https://<your-function>.azurewebsites.net/api/greeting -H "Authorization: Bearer <VALID-ACCESS-TOKEN>"
+Hello, world. You were able to access this because you provided a valid access token with the Greeting.Read scope as a claim.
+```
+
+## About the code
+
+This Azure Function is an anonymous HTTP trigger written in Node.js and uses the built-in [Authentication and authorization in Azure Functions](https://docs.microsoft.com/azure/app-service/overview-authentication-authorization) feature to offload fundamental JWT access token validation. Requests that make it through the built-in authentication feature of Azure Functions are then routed to the Node.js code, which applies additional access token validation checking for a specific scope.
+
+- A missing or invalid (expired, wrong audience, etc) token will result in a `401` response. (Handled by Azure Functions authentication)
+- An otherwise valid token without the proper scope will result in a 403 response.
+- A valid token with the proper scope of `Greeting.Read` will result in the "Hello, world" message.
+
+### Running locally
+
+At the time of this writing, Function app authentication does not support a local development experience that has parity with the on-Azure runtime. You can still execute this locally with `func start` but the authentication functionality provided by the Function app service on Azure will not be invoked; all JWT token validation for authorization (signature, iss, exp, aud) will be skipped.
+
+## Reporting problems
+
+### Sample app not working?
+
+If you can't get the sample working, you've checked [Stack Overflow](http://stackoverflow.com/questions/tagged/msal), and you've already searched the issues in this sample's repository, open an issue report the problem.
+
+1. Search the [GitHub issues](../../../../issues) in the repository - your problem might already have been reported or have an answer.
+1. Nothing similar? [Open an issue](LINK_HERE) that clearly explains the problem you're having running the sample app.
```

Comment:
Can you verify that both of these links are working.  The LINK_HERE needs to be replaced, and I think the "issues" link doesn't feel like it's the right depth of back reference (but could be wrong!)